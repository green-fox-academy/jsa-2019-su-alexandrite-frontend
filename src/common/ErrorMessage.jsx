import React from 'react';
import { View, Text } from 'react-native';
import PropTypes from 'prop-types';
import styles from './styles';

const ErrorMessage = ({ message }) => (
  <View style={styles.errorMessageContainer}>
    <Text style={styles.errorMessageText}>{message}</Text>
  </View>
);


ErrorMessage.propTypes = {
  message: PropTypes.string.isRequired,
};

export default ErrorMessage;
