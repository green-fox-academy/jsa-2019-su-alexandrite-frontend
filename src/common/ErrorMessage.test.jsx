import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from './ErrorMessage';

test('Card rednder properly', () => {
  const tree = renderer.create(
    <ErrorMessage message="Error Message" />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});