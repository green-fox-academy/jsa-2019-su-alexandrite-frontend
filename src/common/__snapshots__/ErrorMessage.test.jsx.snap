// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card rednder properly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff6f6",
      "borderRadius": 6,
      "paddingHorizontal": 8,
      "paddingVertical": 15,
      "width": 690,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#9f3a38",
        "fontWeight": "bold",
      }
    }
  >
    Error Message
  </Text>
</View>
`;
